docs/docs/developer_guide/get-hacking.md

```diff
@@ -64,33 +64,40 @@ to create a `testuser` Keycloak user with the password `tester`.  (You can creat
 ### GitHub App
 
 [Create an OAuth2 application for GitHub](../run_minder_server/config_oauth.md).
-Select `New OAuth App` and fill in the details. The callback URL should be
-`http://localhost:8081/realms/stacklok/broker/github/endpoint`.
```

Comment:
This did not work for me when I ran minder locally. The callback URL in the config_oauth document worked instead.